src/systemctl/systemctl.c

```diff
@@ -4868,6 +4887,9 @@ static int show(int argc, char *argv[], void *userdata) {
                                         return r;
                                 else if (r > 0 && ret == 0)
                                         ret = r;
+
+                                if (r == 4)
+                                        printf("Can't display property %s. Unit %s does not exist.", *patterns, *name);
```

Comment:
errors should be printed with log_error() or such like, not with printf(), especially not to stdout.